OpenSourcePolitics/decidim-module-initiative_state
# Add status

This PR enables status to be used.

You can create, update and destroy status the same way as areas for example, and you can name your status the way you want.
You can then add a status to an initiative when updating it, and this status will be displayed with the tags on the initiative card and on the initiative page.
You can also filter initiatives by status on the initiatives page.

How to test : 

- Go to admin settings
- GO to statuses
- Create a new status
- Go to admin initiatives
- Update a published initiative, and add your status
- Go to the public page of this initiative
- See your status in the tags
- Go to initiatives
- See your status in the initiative card
- Play with status filters to see it works.

For dev reviewers : 
Two erb files could have been defaced but 
- for the form, i wasn't able to find a good selector to insert into
- for the tags, it wasn't detected when calling it from the cell
